action.yml

```
@@ -0,0 +1,36 @@
+name: 'OpenTofu - Setup Tofu'
+description: 'Sets up OpenTofu CLI in your GitHub Actions workflow.'
+author: 'OpenTofu'
+inputs:
+  cli_config_credentials_hostname:
+    description: 'The hostname of a Terraform Cloud/Enterprise instance to place within the credentials block of the OpenTofu CLI configuration file. Defaults to `app.terraform.io`.'
+    default: 'app.terraform.io'
+    required: false
+  cli_config_credentials_token:
+    description: 'The API token for a Terraform Cloud/Enterprise instance to place within the credentials block of the OpenTofu CLI configuration file.'
+    required: false
+  tofu_version:
+    description: 'The version of OpenTofu CLI to install. If no version is given, it will default to `latest`.
```

```suggestion
    description: 'The version of OpenTofu CLI to install. If no version is given, it will default to `latest`.'
```

Missing `'` single-quote at the end of the description.

---

Implemented